solidity/contracts/libraries/staking/Locks.sol

```diff
@@ -0,0 +1,109 @@
+pragma solidity 0.5.17;
+
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "../../KeepRegistry.sol";
+import "./LockUtils.sol";
+
+library Locks {
+    using SafeMath for uint256;
+    using LockUtils for LockUtils.LockSet;
+
+    event StakeLocked(address indexed operator, address lockCreator, uint256 until);
+    event LockReleased(address indexed operator, address lockCreator);
+    event ExpiredLockReleased(address indexed operator, address lockCreator);
+
+    uint256 public constant maximumLockDuration = 86400 * 200; // 200 days in seconds
+
+    struct Storage {
+        // Locks placed on the operator.
+        // `operatorLocks[operator]` returns all locks placed on the operator.
+        // Each authorized operator contract can place one lock on an operator.
+        mapping(address => LockUtils.LockSet) operatorLocks;
+    }
+
+    function lockStake(
```

Comment:
Don't we want to add docs for functions?

Reply:
I am not sure... I had two approaches to this and docs will mostly duplicate what's already in `TokenStaking`. Given that it's a library used only by `TokenStaking`, I think we should be fine with having the documentation just there. WDYT?